pfrl/agents/dqn.py

```diff
@@ -708,6 +719,8 @@ def setup_actor_learner_training(
         update_counter: Optional[Any] = None,
         n_updates: Optional[int] = None,
         actor_update_interval: int = 8,
+        step_hooks: List[Callable[[Optional[env.Env], agent.Agent, int], Any]] = [],
+        optimizer_step_hooks: List[Callable[[Optional[env.Env], agent.Agent, int], Any]] = []
```

Comment:
I wants to clarify this since it is not obvious why `Optional[env.Env]` needs to be there:

This argument would be always `None` for the learner since the learner is not supposed to have the env instance, right? Despite that, by having this argument, we can reuse `pfrl.experiments.hooks.StepHook` as long as it does not use the env argument. While this could be debatable, the decision makes sense to me. Let me know if I miss something.

Reply:
I change the type to `None` because `stephooks` from users are not necessarily subclasses of `pfrl.experiments.hooks.StepHook`. 
https://github.com/pfnet/pfrl/blob/b9d2ee4a2aedd727833dd2aea77d3903ae88050b/pfrl/experiments/hooks.py#L9-L11

Reply:
`None` looks better.